### PR TITLE
Update bind.go

### PR DESCRIPTION
**Pull request by im-kulikov**

### CODE REVIEW

File: web/bind.go [hunk 1]
```diff
@@ -54,14 +59,9 @@ func (b *DefaultBinder) Bind(i interface{}, c Context) (err error) {
...
-func dumpError(err error, logger echo.Logger, req http.Request) {
+func dumpError(err error, logger echo.Logger, dump []byte) {
 	if err != nil {
-		dump, err := httputil.DumpRequest(&req, true)
-		if err != nil {
-			logger.Error(err)
-			return
-		}
-		logger.Debug(string(dump))
+		logger.Debug(string(dump), "test")
```
Comment (pcherednichenko) [inline web/bind.go]: test?

Comment (im-kulikov) [reply web/bind.go]: oh... yeah... debug

Comment (im-kulikov) [reply web/bind.go]: fixed

File: web/bind.go [hunk 2]
```diff
@@ -19,8 +19,13 @@ type (
 
 // Bind implements the `Binder#Bind` function.
 func (b *DefaultBinder) Bind(i interface{}, c Context) (err error) {
-	defer dumpError(err, c.Logger(), *c.Request())
 	req := c.Request()
+	dump, _ := httputil.DumpRequest(req, true)
```
Comment (jenchik) [inline web/bind.go:23]: mb?
```
	defer func(e error){
		dumpError(e, c.Logger(), req)
	}(err)
```

Comment (im-kulikov) [reply web/bind.go:23]: defer works with empty req.Body...

Comment (jenchik) [reply web/bind.go:23]: sorry
```
	defer func(e error){
		dumpError(e, c.Logger(), dump)
	}(err)
```

Comment (im-kulikov) [reply web/bind.go:23]: are you see difference?
```
	dump, _ := httputil.DumpRequest(req, true)

	defer func(){
		dumpError(err, c.Logger(), dump)
	}()
```

Comment (im-kulikov) [reply web/bind.go:23]: what's the point in hiding `err`?

Comment (jenchik) [reply web/bind.go:23]: no, defer always get err equal NIL

Comment (im-kulikov) [reply web/bind.go:23]: you really check this? or only for comment?

Comment (jenchik) [reply web/bind.go:23]: https://play.golang.org/p/nloLcBJcHZo

Comment (im-kulikov) [reply web/bind.go:23]: @jenchik https://play.golang.org/p/1c1J2WFrsVZ

check your self pls...

Comment (jenchik) [reply web/bind.go:23]: ok

**Review by jenchik — DISMISSED** [review]

**Review by pcherednichenko — DISMISSED** [review]

**Review by pcherednichenko — APPROVED** [review]

**Pull request merged**